instrumentation/kamon-scala-future/src/main/resources/reference.conf

```diff
@@ -23,9 +23,9 @@ kamon.instrumentation.futures.scala {
 kanela.modules {
 
   scala-future {
-    name = "Scala Future Intrumentation"
+    name = "Scala Future Instrumentation"
     order = 1
-    description = "Provides automatic context propagation to the thread executing a Scala Future's body and callbacks"
+    description = "Deprecated since Kamon 2.1.21, will be removed in Kamon 2.3.0. Provides automatic context propagation to the thread executing a Scala Future's body and callbacks"
```

Comment:
2.3.0, not 2.2.0?

Reply:
Yeap. We will keep it on Kamon but disabled, then remove it when 2.3.0 comes out.